### DISCUSSION TITLE
Sort-by filter on the main page?

### DISCUSSION THREAD

**Discussion by ghost** [tosdr/CrispCMS#58] [Ideas] [answered]
I've been showing your site to friends lately, and one of the things that's always crossed my mind regarding it is the lack of an A-Z filter for services, and a "sort services by class" bar to let people see which services have been only unrated, or rated-A services only for example.

**Comment by JustinBack** [comment, marked as answer]
I have added the request to the roadmap https://git.tosdr.org/tosdr/roadmap/issues/11
